include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -905,7 +905,7 @@ struct CustomOpApi {
   template <typename T>
   T* GetTensorMutableData(_Inout_ OrtValue* value);
 
-  int8_t GetTensorDeviceType(_In_ const OrtValue* value);
+  const OrtMemoryInfo* GetTensorMemoryInfo(_In_ const OrtValue* value);
```

Comment:
Why is this being added to CustomOpApi?  Shouldn't it be a method on Ort::Value?

Reply:
We use many functions of CustomOpApi in backend_utils.cc so it made sense to me to add there Line no 370
auto tensor_shape = ort.GetTensorTypeAndShape(tensor);
 auto elem_type = ort.GetTensorElementType(tensor_shape);

if ((elem_type == ONNX_TENSOR_ELEMENT_DATA_TYPE_INT64) &&
      (precision == InferenceEngine::Precision::I32)) {
    const int64_t* tensor_data_64 = ort.GetTensorData<int64_t>(tensor);